bin/cide.conf

```diff
@@ -0,0 +1,34 @@
+[DEFAULT]
+# Variable path and stuff to set
+# SRC = "/srv/www/cide.py/src"
+SRC = "/home/local/USHERBROOKE/phiv2201/Projects/cide.py/src"
```

Comment:
Good for now.
A simple fix would be to make this file a template and to generate a conf file from the template.
SRC = "{{SRC_VALUE}}"

and replace all placeholder by a specific value